File: 03-debugging-code/03_Type_Error/spec.rb
describe "add" do
  it "returns a string with 1 and 2 added" do
    add(1,2).should eq("1 + 2 = 3")
  end
  it "returns a string with 5 and 7 added" do
    add(5,7).should eq("5 + 7 = 12")
  end
end

File: 05-if-statements/02_And_Or_2/spec.js
var expect    = require("chai").expect;
var challenge = require("./challenge.js");  

describe("canIGet", function() {
  it("returns true if user wants a computer and has $1,000", function() {
    expect(challenge.canIGet("computer", 1100)).to.equal(true);
  });

  it("returns false for a computer if they don't have $1,000", function() {
    expect(challenge.canIGet("computer", 900)).to.equal(false);
  });

  it("returns true for a iPad if they have $500", function() {
    expect(challenge.canIGet("ipad", 500)).to.equal(true);
  });

  it("returns false for a iPad if they have less than $500", function() {
    expect(challenge.canIGet("ipad", 499)).to.equal(false);
  });
});